### PR TITLE
Update with upstream (supporting polymer 2/hybrid)

### PR DESCRIPTION
**Pull request by tylergraf**
I also changed the polymer version to `1.9 - 2` so it supports both 1 and 2.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Contributing
+
+Looking to contribute to the TreeWeb codebase? You've come to the right place! We welcome pull requests, issues, feedback, etc., and have a few guidelines for contributions:
```
Comment (redbugz) [inline .github/CONTRIBUTING.md]: This file does not belong here

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+A few sentences describing the overall goals of the pull request's commits. Reference any issues/JIRAs/dependent PR's. Should **never** be empty. Review: [CONTRIBUTING.md](/.github/CONTRIBUTING.md)
```
Comment (redbugz) [inline .github/PULL_REQUEST_TEMPLATE.md]: This file does not belong here or should be modified to be more relevant for this type of project.

File: bin/tag_build.sh [hunk 3]
```diff
@@ -0,0 +1,32 @@
+#! /bin/bash
```
Comment (redbugz) [inline bin/tag_build.sh]: This file does not belong here.

File: bower.json [hunk 4]
```diff
@@ -15,18 +15,22 @@
...
-  ],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.2.0",
-    "fetch": "^1.0.0",
-    "promise-polyfill": "polymerlabs/promise-polyfill#^1.0.0",
-    "fetch-mock": "https://unpkg.com/fetch-mock@5.5.0/es5/client-browserified.js"
+    "es6-promise": "stefanpenner/es6-promise#^4.1.0",
+    "fetch": "github/fetch#^2.0.3",
+    "fetch-mock": "https://unpkg.com/fetch-mock@5.10.0/es5/client-browserified.js",
+    "polymer": "Polymer/polymer#1.9 - 2"
```
Comment (redbugz) [inline bower.json]: is there a reason to restrict it to 1.9+? Doesn't it work just fine on older versions of Polymer?

**Comment by tylergraf** [comment]
@redbugz do you know who created the fork? I'm good with using the original repo.

**Pull request closed without merging**